app/src/main/java/org/gem/indo/dooit/views/main/fragments/TipsFragment.java

```diff
@@ -99,8 +104,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
 
         tipsTabAdapter = new TipsTabAdapter(getChildFragmentManager(), getContext(), this);
         viewPager.setAdapter(tipsTabAdapter);
-        TilingDrawable tiled = new TilingDrawable(ContextCompat.getDrawable(getContext(), org.gem.indo.dooit.R.drawable.bkg_clipped));
-        tiled.setTint(getResources().getColor(org.gem.indo.dooit.R.color.light_grey));
+        ShapeDrawable back = new ShapeDrawable();
```

Comment:
Suggest creating a nifty Helper file with a method instead opt copying and pasting code